ipi-java-220-ex/src/main/java/com/ipiecoles/java/java220/Cadre.java

```diff
@@ -0,0 +1,29 @@
+package com.ipiecoles.java.java220;
+
+import org.joda.time.LocalDate;
+
+import java.util.Objects;
+
+public abstract class Cadre extends Employe{

```

Comment:
Attention la classe Cadre n'est pas abstract !
```suggestion
public class Cadre extends Employe{

```